streamalert/alert_processor/outputs/aws.py

```diff
@@ -566,7 +566,8 @@ def _build_email(cls, alert, publication, creds):
         # Attach the record to the email
         if publication.get("@aws-ses.attach_record", True):
             record = json.dumps(alert.record, sort_keys=True, indent=2)
-            msg = cls._add_attachment(msg, "record.json", record)
+            record_name = publication.get("@aws-ses.record_name", "record")
+            msg = cls._add_attachment(msg, f"{record_name}.json", record)
```

Comment:
f-strings are currently not used across StreamAlert. So not sure if the introduction will be accepted or not

Reply:
If this is not the accepted convention then I can easily change this